R/add_data.R

```diff
@@ -299,7 +299,7 @@ addMarkov <- function(dd, transMat, chainLen, wide = FALSE, id = "id",
 
   # if start0lab defined, check that it exists in the transition matrix
   if (!is.null(start0lab)) {
-    if (any(0 < dd[, start0lab, with = FALSE] | dd[, start0lab, with = FALSE] > dim(transMat)[1])) {
+    if (any(0 > dd[, start0lab, with = FALSE] | dd[, start0lab, with = FALSE] > dim(transMat)[1])) {
       c <- condition(c("simstudy::start0probNotInTransMat", "error"),
```

Comment:
Makes sense why it was failing. The fix seems like it should work, but maybe it should be:

```
any(dd[, start0lab, with = FALSE] < 1 | dd[, start0lab, with = FALSE] > dim(transMat)[1]))
```

i.e. if there is a starting value of 0, it should fail.

Reply:
Yes sorry I missed that when I was fixing it the first time, updated it

Reply:
I think it is looking good. I will go through your changes one more time, just to confirm everything else is all good. Can you remind me what additional features you've added other than the new `startProb` argument - I want to make sure our `news` file is updated - you could do that if you want.

Reply:
I just updated genFormula for use with double dot notation, not sure if you've taken a look at that. I think in terms of actually usable additions those are the only two. How do I go about updating the news, I see the file, should I put those things under new features?

Reply:
That would be good.